cheroot/connections.py

```diff
@@ -0,0 +1,254 @@
+"""Utilities to manage open connections."""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import io
+import select
+import six
+import socket
+import time
+
+from . import errors
+from .makefile import MakeFile
+
+from collections import OrderedDict
+
+
+try:
```

Comment:
I feel like this whole try/except cascade should probably go under `_compat` module.

Reply:
No comment on that - I just moved the code from `server.py` to `connections.py`.

Reply:
Yes, but since you're already moving it plz put it under _compat and just import this func from there making this module cleaner.

Reply:
I'm not asking you to change it but I think it doesn't belong here.